src/http-util/http-state/http-state.ts

```diff
@@ -0,0 +1,13 @@
+export namespace HttpState {
+  export const OK = 200;
```

Comment:
enum / namespace 문제를 뒤로 하고 직접 코드 값을 할당하기 보다는
아래처럼 node에 있는 것을 사용하는 것이 좋겠습니다
```suggestion
import { constants } from 'http2';

export namespace HttpState {
  export const OK = constants.HTTP_STATUS_OK;
```